src/layer/S1GRDAWSEULayer.ts

```diff
@@ -38,6 +38,18 @@ export enum Resolution {
   MEDIUM = 'MEDIUM',
 }
 
+export enum SpeckleFilterType {
+  DEFAULT = 'DEFAULT',
```

Comment:
According to the documentation, `DEFAULT` is not one of the possible values of speckle filter type.

Reminder, this library is intended to provide a JS interface for using our APIs, similar to `sentinelhub-py` for Python (https://docs.sentinel-hub.com/api/latest/reference/#operation/process). 

EOB and most other apps of ours happen to use it, but it's made so that it can be used by anyone for any purpose, so it shouldn't have any knowledge of those apps.